src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random random = new Random();
+    private int index;
```

Comment:
make this variable local

Reply:
I made it global, in order not to create new instance of Random class every time we call ColorSupplier() method